### DISCUSSION TITLE
OGPにカスタム性を持たせて欲しい

### DISCUSSION THREAD

**Discussion by kumada15** [increments/qiita-discussions#456] [要望・アイデア・不具合]
■要望の背景
積極的な記事の投稿であったりSNSなどでシェアを促している組織は一定数あると思いますが、現行のOGPですと個人のID+Qiitaアイコンが表示される形となっており、いまいち何処の誰が投稿したのか分かりにくいと感じております。そこでOGPにもQiitaの記事であることが損なわれない範囲でカスタマイズ性があるとうれしいです。

■案
- ガイドラインを設けてもらい、それに則っていればフルにカスタマイズ可能とする
- 幾つかのテンプレートを用意してもらい、ユーザ或いはOrganizationで選択出来るようにする
- 既存のものにOrganizationアイコンかOrganization名を記載、など

**Comment by tanaka-asami415** [comment]
@kumada15 
ご意見ありがとうございます🙏
現在の仕組みだとOGP作成時にCDNを利用するコストが発生しているため
カスタム性を持たせる場合、更にコストがかかってしまうため、しばらくは対応が難しいです🙏
アイディアとしては面白いと思うので、将来的には検討したいと思います！

**Reply by kumada15** [reply]
@tanaka-asami415 
ご返答ありがとうございます。
ZennのPublicationが対応されていて、いいなと思ったので是非ご検討いただけると幸いです。
